pypeit/images/buildimage.py

```diff
@@ -26,7 +26,9 @@ class ArcImage(pypeitimage.PypeItImage):
     version = pypeitimage.PypeItImage.version
 
     # I/O
-    output_to_disk = ('ARC_IMAGE', 'ARC_FULLMASK', 'ARC_DETECTOR')
+    output_to_disk = ('ARC_IMAGE', 'ARC_FULLMASK', 'ARC_DETECTOR',
+                      'ARC_DET_IMG', # For echelle multi-detector wavelengths
```

Comment:
I considered that, but to enable `WaveCalib`
to generate the wavelength image it needs
this and it may not have the rest of what is needed
to regenerate.